### PR TITLE
Remove the term whitelist

### PR DESCRIPTION
**Pull request by sameerz**
1. Please write a description in this text box of the changes that are being
   made.

Removing the term whitelist per https://tools.ietf.org/id/draft-knodel-terminology-00.html

2. Please ensure that you have written units tests for the changes made/features
   added.

No new tests, ran `mvn clean verify -DskipTests` to verify

### CODE REVIEW

**Comment by sameerz** [comment]
Related to https://github.com/NVIDIA/spark-rapids/issues/363

**Comment by sameerz** [comment]
build

File: integration_tests/src/main/python/hash_aggregate_test.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/enable
```
Comment (revans2) [inline integration_tests/src/main/python/hash_aggregate_test.py:1]: I think this is a type that is causing the build to fail

Comment (sameerz) [reply integration_tests/src/main/python/hash_aggregate_test.py]: Apologies, I realized after the build failed that I ran with -DskipTests on a non-GPU machine.  I reran with `mvn clean verify -Dcuda.version=cuda10-2` and the tests passed.  I will try rerunning the premerge tests.

**Comment by sameerz** [comment]
build

**Pull request merged**